nipype/interfaces/tests/test_io.py

```diff
@@ -407,6 +407,13 @@ def test_freesurfersource():
     assert fss.inputs.subjects_dir == Undefined
 
 
+def test_freesurfersource_incorrectdir():
+    fss = nio.FreeSurferSource()
+    with pytest.raises(Exception) as err:
```

Comment:
Doesn't it work using `with pytest.raises(TraitError)` ?

Reply:
you're right, after importing `TraitError` it works...